File: solutions/go/rna-transcription/1/rna_transcription.go
package strand

import (
	"strings"
)

var RNAtoDNA = map[rune]rune{
	'G': 'C',
	'C': 'G',
	'T': 'A',
	'A': 'U',
}

func ToRNA(dna string) string {
	var result strings.Builder
	for _, dnaRune := range dna {
		if dnaRune, ok := RNAtoDNA[dnaRune]; ok {
			result.WriteRune(dnaRune)
		}
	}

	return result.String()
}

File: solutions/python/rna-transcription/1/rna_transcription.py
DNA_TO_RNA = {
    "G": "C",
    "C": "G",
    "T": "A",
    "A": "U"
}

def to_rna(dna_strand):
    if len(dna_strand) == 0:
        return ""

    rna_strand = ""
    for i in range(0, len(dna_strand)):
        rna_strand += DNA_TO_RNA[dna_strand[i]]

    return rna_strand
